redhat-cop/openshift-applier
# Add task file and tests to support kustomize templates with apply -k

#### What does this PR do?
Add support for applying kustomize templates.

#### How should this be tested?
Check out PR branch and run:

```
ansible-playbook -i tests/inventories/kustomize/ playbooks/openshift-cluster-seed.yml
```

Then check `oc get cm -n oa-ci-kustomize-template-dev` so see the generated configmap

#### Is there a relevant Issue open for this?
n/a

#### Who would you like to review this?
cc: @redhat-cop/openshift-applier
